velodyne_msgs/package.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
```

Comment:
If you're going to use package format 3 here, please do so with all of the other packages.

Reply:
Will update the others.